crates/requirements-txt/src/lib.rs

```diff
@@ -590,19 +589,18 @@ fn parse_entry(
     } else if s.eat_if("-c") || s.eat_if("--constraint") {
         let constraints_file = parse_value(content, s, |c: char| !['\n', '\r', '#'].contains(&c))?;
         let end = s.cursor();
-        eat_trailing_line(content, s)?;
```

Comment:
Not necessary, I think, because the next step after parsing an entry is to eat the trailing line.